### ISSUE TITLE
Incorrect line number for `Z110`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#392]
# Bug report

## What's wrong

Consider this example:

```python
    @classmethod  # lineno: 95
    def from_token(
        cls: Type[TAssign],
        token: lex.LexToken,
        equal: lex.LexToken = None,
        value: lex.LexToken = None,
    ) -> TAssign:
```

Report: `dotenv_linter/grammar/fst.py:95:5: Z110 Found wrong variable name: value`

## How is that should be

It should report `value: lex.LexToken = None,` as an incorrect line.
